usr/iscsiadm.c

```diff
@@ -2560,18 +2657,17 @@ static int verify_node_params(struct list_head *params, struct node_rec *rec)
 }
 
 /* TODO cleanup arguments */
-static int exec_node_op(int op, int do_login, int do_logout,
-			int do_show, int do_rescan, int do_stats,
+static int exec_node_op(struct iscsi_context *ctx, int op, int do_login,
+			int do_logout, int do_show, int do_rescan, int do_stats,
 			int info_level, struct node_rec *rec,
 			struct list_head *params)
 {
 	int rc = 0;
 
-	if (rec)
```

Comment:
The rec value can be null, so you can't remove this check.

Reply:
Thanks. Then I will fix the NULL deference at
```
	if ((!do_login && !do_logout && op == OP_NOOP) &&
	    (!strlen(rec->name) && !strlen(rec->conn[0].address) &&
	     !strlen(rec->iface.name))) {
```